tests/extensions/helpers.py

```diff
@@ -0,0 +1,9 @@
+from typing import Dict, Union
```

Comment:
I'm not sure if it's proper file name, in other modules we use `utils.py`(eg. `tests/api/utils.py`).